_data/info.yml

```diff
@@ -0,0 +1,22 @@
+contacts:
```

Comment:
We're using `_data/` folder for data not configs. Usually, in a jekyll project all the configs are stored in the same place, i.e in `_config.yml`. 
This has the benefit of not having to look around for configs in other places. 

You probably shouldn't add `url` and `class` into `_config.yml` either as they're basically html. So this can probably be solved in other ways?

Reply:
> Usually, in a jekyll project all the configs are stored in the same place, i.e in _config.yml.

That totally depends on the person. I was seeing social contact links as data rather than config.
Anyway, following one of the most popular Jekyll project way of handling social contacts

https://github.com/mmistakes/minimal-mistakes/blob/f7770ad0799b1c847ea35972f1d286e71a1a50ee/_config.yml#L101-L119